runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/process/actions/excel.ts

```diff
@@ -35,6 +35,7 @@ const excelActionsTranslations: typeof englishExcelActionsTranslations = {
     'Process.Details.Modeler.Actions.Excel.RenameWorksheet.Label': 'Zmień nazwę arkusza',
     'Process.Details.Modeler.Actions.Excel.RenameWorksheet.NewName': 'Nowa nazwa',
     'Process.Details.Modeler.Actions.Excel.RenameWorksheet.Worksheet.Info': 'Pozostawienie pustego pola spowoduje zmianę nazwy aktywnego arkusza',
+    'Process.Details.Modeler.Actions.Excel.WorksheetExists.Label': 'Czy arkusz istnije',
```

Comment:
Imo it should have actual action included, so "Sprawdź czy arkusz istnieje". same reason - naming convention for actions (do something)